packages/oui-select-picker/_mixins.less

```diff
@@ -109,29 +62,6 @@
         }
       }
     }
-
-    // sub components exceptions
-    .oui-select,
-    .oui-ui-select-container {
-      margin-bottom: 0;
-    }
-
-    .oui-select > .oui-icon {
-      background-color: inherit;
-    }
-
-    .oui-ui-select-container {
-      .ui-select-match {
-        margin-bottom: 0;
-        text-align: left;
-        width: 100%;
-      }
-
-      .ui-select-match,
-      .ui-select-choices {
-        min-width: 100%;
-      }
-    }
```

Comment:
Code deleted in this file is transfered into _normalize.less. 
Only code that sets color on component is kept